crates/ironrdp/Cargo.toml

```diff
@@ -53,9 +53,10 @@ ironrdp-blocking.workspace = true
 ironrdp-cliprdr-native.workspace = true
 anyhow = "1"
 async-trait = "0.1"
-bmp = "0.5"
+image = { version = "0.25.5", default-features = false, features = ["png"] }
```

Comment:
`image` is a rather big crate that I would rather not use just for outputting an image file in an example. (Even if we don’t enable all the features.) What is the reason for switching to `image` and outputting a png instead?

`bmp` may not be updated anymore, but it only depends on a single dependency which is not outdated (no new major bump), and it’s just about outputting a bmp file so I wouldn’t surprised if the crate was considered complete.

If you really want to switch to png images, what about using the [`png` crate](https://lib.rs/crates/png) directly as we don’t really need all the features of the `image` crate?